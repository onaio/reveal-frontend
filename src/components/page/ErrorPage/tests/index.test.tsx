import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import React from 'react';
import { ErrorPage } from '..';

describe('src/components/pages/errorPage', () => {
  it('renders correctly', () => {
    const wrapper = mount(<ErrorPage />);
    expect(toJson(wrapper)).toMatchSnapshot('default props');
  });
  it('renders correctly with props', () => {
    const props = {
      errorMessage: 'Something is about to hit the fan',
    };
    const wrapper = mount(<ErrorPage {...props} />);
    expect(toJson(wrapper)).toMatchSnapshot('default props');
  });
});
