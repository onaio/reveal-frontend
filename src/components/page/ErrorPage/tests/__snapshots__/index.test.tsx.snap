// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/pages/errorPage renders correctly with props: default props 1`] = `
<ErrorPage
  errorMessage="Something is about to hit the fan"
>
  <div
    className="global-error-container"
  >
    <p>
      An error ocurred. Please try and refresh the page.
    </p>
    <p>
      The specific error is
      : 
      <span
        className="text-danger"
      >
        Something is about to hit the fan
      </span>
    </p>
  </div>
</ErrorPage>
`;

exports[`src/components/pages/errorPage renders correctly: default props 1`] = `
<ErrorPage
  errorMessage=""
>
  <div
    className="global-error-container"
  >
    <p>
      An error ocurred. Please try and refresh the page.
    </p>
  </div>
</ErrorPage>
`;
